plugins/syntax/int63_syntax.ml

```diff
@@ -20,8 +20,8 @@ open Libnames
 
 (*** Constants for locating int63 constructors ***)
 
-let q_int63 = qualid_of_string "Coq.Numbers.Cyclic.Int63.PrimInt63.int"
-let q_id_int63 = qualid_of_string "Coq.Numbers.Cyclic.Int63.PrimInt63.id_int"
+let q_uint63 = qualid_of_string "Coq.Numbers.Cyclic.Int63.PrimInt63.int"
+let q_id_uint63 = qualid_of_string "Coq.Numbers.Cyclic.Int63.PrimInt63.id_int"
```

Comment:
No need for this change anymore since signed int is now just a notation alias to PrimInt63.int.

Reply:
Nevermind, this files disappears.

Reply:
BTW, why is it still there?

Reply:
No idea, I'll just remove it completely.